requirements/requirements.txt

```diff
@@ -1,7 +1,9 @@
-bottleneck>=1.3.4
-freud-analysis>=2.7.0
-kneed>=0.8.2
-numpy>=1.21
-ruptures>=1.1.5
-scikit-learn>=0.24.0
-pandas>=1.1.3
+bottleneck
+freud-analysis
+kneed
+numba
+numpy
+ruptures
+scikit-learn
+pandas
+xarray
```

Comment:
shouldn't these have lowest supported version?

Reply:
Probably - I will find the lowest variants for each.

Reply:
You can probably reuse some of the old ones?